install/kubernetes/cilium/templates/spire/server/configmap.yaml

```diff
@@ -0,0 +1,70 @@
+{{- if and .Values.auth.mTLS.spire.enabled .Values.auth.mTLS.spire.install.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: spire-server
+  namespace: {{ .Values.auth.mTLS.spire.install.namespace }}
+data:
+  server.conf: |
+    server {
+      bind_address = "0.0.0.0"
+      bind_port = "8081"
+      socket_path = "/tmp/spire-server/private/api.sock"
+      trust_domain = {{ .Values.auth.mTLS.spire.trustDomain | quote }}
+      data_dir = "/run/spire/data"
+      log_level = "INFO"
+      #AWS requires the use of RSA.  EC cryptography is not supported
+      ca_key_type = "rsa-2048"
+
+      ca_subject = {
+        country = ["US"],
+        organization = ["Cilium SPIFFE"],
+        common_name = "SPIRE",
+      }
+
+      admin_ids = [
+        "spiffe://{{ .Values.auth.mTLS.spire.trustDomain }}/{{ .Values.auth.mTLS.spire.identities.ciliumAgent }}",
```

Comment:
why does the agent need admin permissions? This would enable every attested agent to be able to create identities at will

Reply:
Agreed, I don't think the agent should need admin permissions.

Reply:
ah good point :100: